### DISCUSSION TITLE
Anyone from Online MSCS programs successfully get the benefits?

### DISCUSSION THREAD

**Discussion by lottepy** [community/community#56323] [GitHub Education]
### Hi everyone,

As online student don't have student card, I wonder what do you guys upload?

**Comment by samus-aran** [comment]
Hi @lottepy 

There is multiple accepted pieces of evidence accepted: Academic email; Transfer Script; Portal Screenshot etc. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
